firebase-firestore/src/main/java/com/google/firebase/firestore/SetOptions.java

```diff
@@ -59,7 +59,8 @@ public FieldMask getFieldMask() {
 
   /**
    * Changes the behavior of {@code set()} calls to only replace the values specified in its data
-   * argument. Fields omitted from the {@code set()} call will remain untouched.
+   * argument. Fields omitted from the {@code set()} call will remain untouched. If your input sets
+   * any field to an empty map, all nested fields at this location are overwritten.
```

Comment:
optional: use `{@code Map}?

Reply:
I left it as is as it creates weird capitalization and should also really be a `{@link Map}`, which then is too generic to be useful.